cookbook/attributes/default.rb

```diff
@@ -25,6 +25,12 @@
 default['redisio']['package_install'] =
   Chef::VersionConstraint.new('>= 14.04').include?(node['platform_version'])
 
+# Set to true if redis-server should be installed
+default['redis']['local'] = true
+
+# Set to true if mysql should be installed
+default['database']['local'] = true
+
```

Comment:
Should these default to `false`?  They comment in the `recipes/database.rb` makes me think so.

"`XXX: This configuration is currently for testing purposes only!`"